R/validate_data.R

```diff
@@ -806,5 +809,35 @@ validate_variable_mapping <- function(data.list) {
 
 
 
-
+# Check mapped_id
+# 
+# @param data.list
+#     (list of data frames) A named list of data frames, each of which is an 
+#     ecocomDP table.
+#
+# @return 
+#     (character) If mapped_id does not resolve with a status of 200, then a message is returned.
+#
+validate_mapped_id <- function(data.list) {
+  
+  if ("variable_mapping" %in% names(data.list)) {
+    message("  mapped_id")
+    if (!curl::has_internet()) return("Variable mapping. Mapped_ids can not be validated offline")
```

Comment:
@kzollove This may be something we want to add to the entire code base later, but for now remove it.